modules/tinymce/src/themes/silver/main/ts/backstage/TooltipsBackstage.ts

```diff
@@ -0,0 +1,61 @@
+import { AlloyComponent, GuiFactory, TooltippingTypes } from '@ephox/alloy';
+import { Fun, Result } from '@ephox/katamari';
+
+export interface TooltipsProvider {
+  readonly getConfig: (spec: { tooltipText: string }) => TooltippingTypes.TooltippingConfigSpec;
+}
+
+export const TooltipsBackstage = (
+  getSink: () => Result<AlloyComponent, any>
+): TooltipsProvider => {
+
+  const tooltipDelay = 800;
+
+  let numActiveTooltips = 0;
+
+  const alreadyShowingTooltips = () => numActiveTooltips > 0;
+
+  const getConfig = (spec: { tooltipText: string }) => ({
+    delayForShow: () => alreadyShowingTooltips() ? 0 : tooltipDelay,
```

Comment:
This probably shouldn't be `0`. I think I misunderstood the original comment. But it's just supposed to be (noticeably) lower than the normal delay.